{{cookiecutter.repo_name}}/package.json

```diff
@@ -1,47 +1,56 @@
 {
-  "name": "{{cookiecutter.repo_name}}",
+  "name": "{{ cookiecutter.repo_name }}",
   "version": "0.1.0",
-  "private": true,
+  "description": "{{ cookiecutter.description }}",
+  "scripts": {
+    "postinstall": "gulp build:production",
+    "build": "gulp build:production",
+    "start": "gulp start"
+  },
+  "author": "{{ cookiecutter.author }}",
   "dependencies": {
-    "bower": "^1.6.5",
-    "browser-sync": "^2.0.0",
-    "browserify": "^8.1.3",
-    "del": "^1.1.1",
-    "event-stream": "^3.2.2",
-    "gulp": "^3.8.8",
-    "gulp-autoprefixer": "^2.1.0",
-    "gulp-concat": "^2.4.2",
-    "gulp-filter": "^2.0.1",
-    "gulp-htmlmin": "^0.2.0",
-    "gulp-if": "^1.2.5",
-    "gulp-imagemin": "^2.2.0",
-    "gulp-jshint": "^1.9.0",
-    "gulp-load-plugins": "^0.8.0",
-    "gulp-minify-css": "^0.4.5",
-    "gulp-modernizr": "1.0.0-alpha",
-    "gulp-notify": "^2.2.0",
-    "gulp-plumber": "^0.6.6",
-    "gulp-replace": "^0.5.0",
-    "gulp-rev-all": "^0.7.1",
-    "gulp-sass": "^2.0.4",
-    "gulp-shell": "^0.3.0",
-    "gulp-sourcemaps": "^1.3.0",
-    "gulp-swig": "^0.7.3",
-    "gulp-task-listing": "^1.0.0",
-    "gulp-uglify": "^1.1.0",
-    "gulp-util": "^3.0.1",
-    "gumshoe": "git://github.com/cferdinandi/gumshoe",
-    "jshint-stylish": "^1.0.0",
-    "lazypipe": "^0.2.2",
-    "main-bower-files": "^2.5.0",
-    "minimist": "^1.1.0",
-    "run-sequence": "^1.0.1",
-    "shelljs": "^0.3.0",
-    "swig": "^1.4.2",
-    "vinyl-source-stream": "^1.1.0"
+    "autoprefixer": "6.1.0",
+    "babel-preset-es2015": "6.1.4",
+    "babelify": "7.2.0",
+    "browser-sync": "2.10.0",
+    "browserify": "12.0.1",
+    {% if cookiecutter.use_foundation_sites == 'y' -%}"browserify-shim": "3.8.12",{%- endif %}
+    "del": "2.0.2",
+    {% if cookiecutter.use_foundation_sites == 'y' -%}"foundation-sites": "6.1.2",{%- endif %}
+    "gulp": "3.9.0",
+    "gulp-concat": "2.6.0",
+    "gulp-cssnano": "2.1.0",
+    "gulp-if": "2.0.0",
+    "gulp-postcss": "6.0.1",
+    "gulp-sass": "2.1.0",
+    "gulp-sourcemaps": "1.6.0",
+    "gulp-uglify": "1.4.2",
+    "gulp-util": "3.0.6",
+    {% if cookiecutter.use_foundation_sites == 'y' -%}"jquery": "2.1.4",{%- endif %}
+    "run-sequence": "1.1.4",
+    "vinyl-buffer": "1.0.0",
+    "vinyl-source-stream": "1.1.0"
   },
-  "scripts": {
-    "postinstall": "bower install && gulp build",
-    "start": "gulp build"
+  "devDependencies": {
+    "watchify": "3.6.0"
+  },
+  {% if cookiecutter.use_foundation_sites == 'y' -%}
+  "browser": {
+    "jquery": "./node_modules/jquery/dist/jquery.js",
+    "foundation": "./node_modules/foundation-sites/js/foundation.js"
+  },
+  "browserify-shim": {
+    "jquery": "$",
+    "foundation": "foundation"
+  },
+  "browserify": {
+    "transform": [
+      "browserify-shim"
+    ]
+  },
+  {%- endif %}
+  "engines": {
+    "npm": ">=3.0.0",
+    "node": ">=4.2.0"
```

Comment:
Since the `.nvmrc` defined the node version at 4.2.1, I think we should lock this version to the same value.